app/medInria/medFilteringWorkspace.cpp

```diff
@@ -48,34 +48,31 @@ medFilteringWorkspace::~medFilteringWorkspace()
  */
 void medFilteringWorkspace::setupViewContainerStack()
 {
-    if ( !this->stackedViewContainers()->count() )
-    {
-        d->inputContainer = this->stackedViewContainers()->addContainerInTab(this->name());
-        QLabel *inputLabel = new QLabel("INPUT");
-        inputLabel->setAlignment(Qt::AlignCenter);
-        d->inputContainer->setDefaultWidget(inputLabel);
-
-        d->inputContainer->setClosingMode(medViewContainer::CLOSE_VIEW);
-        d->inputContainer->setUserSplittable(false);
-        d->inputContainer->setMultiLayered(false);
-
-        d->outputContainer = d->inputContainer->splitVertically();
-        QLabel *outputLabel = new QLabel("OUTPUT");
-        outputLabel->setAlignment(Qt::AlignCenter);
-        d->outputContainer->setDefaultWidget(outputLabel);
-        d->outputContainer->setClosingMode(medViewContainer::CLOSE_VIEW);
-        d->outputContainer->setUserSplittable(false);
-        d->outputContainer->setMultiLayered(false);
-        d->outputContainer->setUserOpenable(false);
-
-        connect(d->inputContainer, SIGNAL(viewContentChanged()), this, SLOT(changeToolBoxInput()));
-        connect(d->inputContainer, SIGNAL(viewRemoved()), this, SLOT(changeToolBoxInput()));
-
-        this->stackedViewContainers()->lockTabs();
-        this->stackedViewContainers()->hideTabBar();
```

Comment:
These 2 lines (74-75) are useless now. We call at the beginning `medAbstractWorkspace::setupViewContainerStack()`